packages/loader/container-loader/src/utils.ts

```diff
@@ -68,6 +69,7 @@ function convertSummaryToSnapshotWithEmbeddedBlobContents(
                 treeNode.blobs[blobId] = typeof summaryObject.content === "string" ?
                     fromUtf8ToBase64(summaryObject.content) :
                     bufferToString(summaryObject.content, "base64");
+                blobs[blobId] = stringToBuffer(treeNode.blobs[blobId], "base64");
```

Comment:
Is treeNode.blobs[blobId] left for compatibility with old runtime mostly? I.e. eventually it should be removed, right?
If so, should we write code such that it's easy to remove, i.e. initialize blobs first (not through doing double-conversion) and then convert it to treeNode.blobs, such that last step can be easily removed later?

Reply:
Yes, keeping this for old runtime compatibility, It will removed eventually. Moving it higher.